src/main/java/net/minestom/server/network/packet/server/play/EntitySoundEffectPacket.java

```diff
@@ -9,15 +10,20 @@
 public class EntitySoundEffectPacket implements ServerPacket {
 
     public int soundId;
-    public SoundCategory soundCategory;
+    public Sound.Source soundSource;
     public int entityId;
     public float volume;
     public float pitch;
 
+    /**
+     * @deprecated Use {@link #soundSource}
+     */
+    @Deprecated public SoundCategory soundCategory;
```

Comment:
Same as JsonMessages in earlier comment: either keep Minestom type or delete it completly. 
But in this case, I prefer keeping SoundCategory, but I am open to debate.